src/ServiceControl.Persistence.RavenDb5/ErrorMessagesDataStore.cs

```diff
@@ -601,7 +599,7 @@ public async Task<(string[] ids, int count)> UnArchiveMessagesByRange(DateTime f
                            update
                            {{
                                 msg.Status = args.Unresolved
-                                delete msg['@metadata']['@expires']
+                                {ExpirationManager.DeleteExpirationFieldScript}
```

Comment:
From the examples on https://ravendb.net/docs/article-page/6.0/csharp/client-api/operations/patching/set-based it looks like the statements inside the `update` block are going to need semicolons